ParkingZone/appsettings.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Server=SARDOR\\SQLEXPRESS;Database=ParkingZone;Trusted_Connection=True;MultipleActiveResultSets=true;Encrypt=false"
+    "DefaultConnection": "Server=SARDOR\\SQLEXPRESS;Database=Parking-Zone;Trusted_Connection=True;MultipleActiveResultSets=true;Encrypt=false"
```

Comment:
Why did you switch the database? Never do so! On real projects, you won't have such opportunity. Therefore switch back to previous db and continue working there.